src/wallet/reconstruct_seed_flow.c

```diff
@@ -129,8 +129,18 @@ reconstruct_state_e reconstruct_seed_handler(reconstruct_state_e state,
 /*****************************************************************************
  * STATIC FUNCTIONS
  *****************************************************************************/
+void get_seed_from_secret(uint8_t *secret, uint8_t *seed_out) {
```

Comment:
Rather than breaking this into a completely different function.. can we split the state of the `reconstruct_seed_handler` - one in which mnemonics are generated and other in which seed is calculated using passphrase.. 

So something similar to `init_state` in `handle_secret_reconstruction` can be applied to `exit_state`

Reply:
In that case, `PASSPHRASE_INPUT` state should also not be in same handler. It is introducing an inconsistency, where we have flexibility to have configurable init state but not configuration exit state